alacritty/src/config/bindings.rs

```diff
@@ -365,11 +346,26 @@ macro_rules! bindings {
     }};
 }
 
+macro_rules! trigger {
+    (KeyBinding, $key:literal,) => {
+        BindingKey::Keycode { key: Character($key.into()), location: KeyLocation::Standard }
+    };
+    (KeyBinding, $key:literal, $location:expr) => {
+        BindingKey::Keycode { key: Character($key.into()), location: $location }
+    };
+    (KeyBinding, $key:expr,) => {
+        BindingKey::Keycode { key: $key, location: KeyLocation::Standard }
+    };
+    ($ty:ident, $key:expr,) => {
+        $key
+    };
+}
```

Comment:
Why is this a separate macro?

Reply:
You can try to make it not a separate macro, but for me it's way more readable than what was before.

I've tried to encode all that special case handling into our main macro, but it was not always working because it was matching `Character("l")` in other branch after the first invocation and so on.

Reply:
I made some minor adjustments (mainly wrapping everything in double `{{ }}` but the alternatives I could come up with were less readable than this solution.

The best way to simplify these macros would likely be to switch away from `bindings!` and go for `binding!`, but I'm not interested in doing that right now and I'm sure neither are you.